<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="localhost" name="Default test" tests="1" failures="1" timestamp="3 Aug 2016 13:02:15 GMT" time="44.519" errors="0">
  <testcase name="main" time="28.695" classname="regressioncases.AllModule">
    <failure type="org.junit.ComparisonFailure" message="expected:&amp;lt;[db14a98b-295f-43ba-a108-2fd6ac48df82]@it.com&amp;gt; but was:&amp;lt;[5f4709ae-7f6f-4975-854b-140d721e61f1]@it.com&amp;gt;">
      <![CDATA[org.junit.ComparisonFailure: expected:<[db14a98b-295f-43ba-a108-2fd6ac48df82]@it.com> but was:<[5f4709ae-7f6f-4975-854b-140d721e61f1]@it.com>
	at org.junit.Assert.assertEquals(Assert.java:115)
	at org.junit.Assert.assertEquals(Assert.java:144)
	at uqmoduletestcases.UQDeckShareCases.verifyRecordInDatabase(UQDeckShareCases.java:377)
	at uqmoduletestcases.UQDeckShareCases.verifySendInviteNonUQPeople(UQDeckShareCases.java:97)
	at uqmoduletestcases.UQDeckShareCases.verifyAllDeckShareCases(UQDeckShareCases.java:423)
	at regressioncases.AllModule.main(AllModule.java:126)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
]]>
    </failure>
  </testcase> <!-- main -->
</testsuite> <!-- Default test -->
